### PR TITLE
OS specific install

### PR DESCRIPTION
**Pull request by JerryHamm**
Using the Python `platform module and distro package`,  the `wx install surface` command for installing the Surface CDMS will only work on Linux machines. closing #4

### CODE REVIEW

File: weathereye/cli.py [hunk 1]
```diff
@@ -1,16 +1,44 @@
...
+    """Command to install SURFACE CDMS"""
+
+    # check if OS is supported before install
+    host_system = system().lower()
+    host_name = id()
+    host_version = version()
+
+    if host_system != 'linux':
+        
+        click.echo(f"{("macOS" if host_name == "darwin" else host_name), host_version} is not supported for surface CDMS")
```
Comment (isedwards) [inline weathereye/cli.py:33]: I get an error here:
```
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/codespace/.local/lib/python3.10/site-packages/weathereye-0.1.0-py3.10.egg/weathereye/cli.py", line 33
    click.echo(f"{("macOS" if host_name == "darwin" else host_name), host_version} is not supported for surface CDMS")
                    ^^^^^
SyntaxError: f-string: unmatched '('```
```

Try changing the outer quotes from double quotes (") to single (')?

**Review by isedwards — CHANGES_REQUESTED** [review]
@JerryHamm

File: weathereye/cli.py (hunk 1 above)
Comment (isedwards) [reply weathereye/cli.py:33]: Perhaps you could also add "currently", e.g.
"is not **currently** supported for SURFACE CDMS"

Comment (JerryHamm) [reply weathereye/cli.py]: The problem should be solved now Ian.

**Review by isedwards — APPROVED** [review]

**Pull request merged**